docs/installation/getting-started/maintenance.md

```diff
@@ -12,7 +12,58 @@ modified_at: 2021-07-16
+
+It is good to follow [CKEditor Ecosystem Blog](https://ckeditor.com/blog/) as it also brings other important articles and news about features, changes and planned development.
+
+### Migration guides
+
+Should there be any breaking or important changes that affect the editor code and require special attention, these will also be published in CKEditor 5 documentation in the {@link updating/index Updating section}. These guides will provide a more technical, code-oriented information directed at integrators, administrators and developers and will offer solutions and necessary steps to take while updating.
+
+Administrators and developers should always refer to migration guides after each release and make sure all to implement all the introduced changes properly to ensure stable and uninterrupted operation.
+
+## Daily maintenance
+
+### Upgrade regularly
+
+CKEditor 5 should be upgraded frequently as bug fixes and new features, including support for new browser versions, are not backported. While installing and using CKEditor 5 instance, especially when adding new features, always make sure all the packages are of the same (preferably latest) version. If this requirement is not met, errors may occur.
+
+If you modified any distribution files to add your custom configuration, pay special attention to not overwrite your content (including config.js, contents.css, styles.js and custom template files) when upgrading.
+
+### Use minified versions
+
+You should always use minified CKEditor 5 versions (official releases, optimized builds) in production environments. The development, source code version is only suitable for tests and should not be used in production environment.
+
+### Update your custom builds
+
+If you want to skip some editor features, customize your build with online builder and remove unneeded functionality. Alternatively  create your own customized build from scratch. It is a bad practice to download a full editor package and then remove plugins or buttons in your configuration. You will only be loading unnecessary stuff without any good reason.
+
+### Use online builder to add plugins
+Some releases would bring new features and new plugins and replace old one and make them obsolete. If you want to install additional plugins, use online builder instead of adding them manually.
+
+This will reduce the risk of omitting plugin dependencies and in the created optimized build all plugins will be merged into one file, which will reduce the time needed to load them.
+
+## Safety
+
+Observe any security alerts that are published by the CKEditor 5 team and always act promptly to apply patches and upgrades. Keeping your editor up-to-date is crucial to the security and integrity of you content and data. If you are using framework integrations always follow any information provided by framework developers, too.
```

Comment:
It might be a good idea to link to our [GitHub security advisories](https://github.com/ckeditor/ckeditor5/security/advisories) while mentioning security.

Reply:
Addressed in the commit.